root/user/edits.tt

```diff
@@ -1,29 +1,29 @@
-[%- param = { name => link_editor(user) };
+[%- param = { editor => link_editor(user), username => user.name  };
     SWITCH c.action.name;
         CASE 'votes';
-            page_title = l('Votes');
-            page_heading = l('Votes by {name}', param);
+            page_title = l('Votes by {username}', param);
+            page_heading = l('Votes by {editor}', param);
         CASE 'open';
-            page_title = l('Open Edits');
-            page_heading = l('Open Edits by {name}', param);
+            page_title = l('Open edits by {username}', param);
+            page_heading = l('Open edits by {editor}', param);
```

Comment:
How come for all of these, one string uses `username` for the var arg and the other `editor`? I'd think `editor` makes perfect sense for both (doesn't really make a difference that one's clickable and one's not).

Otherwise, I support eradicating bizarre capitalization.

Reply:
Because how do I pass a different argument to each then? (If I pass editor to the title, it shows it as an HTML a element)

Reply:
By having two vars like
```
[%- title_param = { editor => user.name };
    heading_param = { editor => link_editor(user) };
    SWITCH c.action.name;
        CASE 'votes';
            page_title = l('Votes by {editor}', title_param);
            page_heading = l('Votes by {editor}', heading_param);
```